crates/ansi-to-html/src/color.rs

```diff
@@ -81,6 +84,41 @@ impl Color {
             }
         })
     }
+
+    pub(crate) fn into_opening_fg_span(self, var_prefix: &Option<String>) -> String {
```

Comment:
`&Option<String>` is an unnecessary indirection. Please use `Option<&str>` instead (which you can convert to with `Option::as_deref()`).